### ISSUE TITLE
New Alpha Release?

### ISSUE THREAD

**Issue by Chaostheorie** [bigbluebutton/bigbluebutton#11254]
I've been wondering when the next alpha release will be published. The last release, as of writing 2.3-alpha-4, was over a month ago and since then a lot of promising features have been implemented. Especially the patch for a confirmation when ending a meeting is in high demand on a big blue button instance I'm working with. 

It would be nice to know if the release, if not planned explicitly, is to be expected next month or later. 

Thank you for any response in advance. Your work is helping a lot and contributes as lot to keeping up with the current situation.

**Comment by basisbit** [comment]
For support requests, please use the [forum / mailing list](https://groups.google.com/forum/#!forum/bigbluebutton-dev). This issue tracker here is only for BBB development related issues. Sorry.

TLDR: next release will be published when developers think it is helpful to get further feedback from testers. BBB 2.3 is not yet usable for production, the only purpose of new alpha/beta releases is to get new tester feedback.
